src/app/zap-templates/zcl/zcl.json

```diff
@@ -811,11 +811,7 @@
             "MaxClientCertificates"
         ],
         "TLS Client Management": ["ClusterRevision", "MaxProvisioned"],
-        "Thermostat": [
```

Comment:
Not sure why zap generation removed this. These are intended to be handled by the attribute access interface and the code is written that way. is this a zap bug? I didn't want to manually edit this myself. @ReneJosefsen @hasty any comments?